packages/app-project/src/hooks/useComments.js

```diff
@@ -11,10 +9,8 @@ const SWROptions = {
   refreshInterval: 0
 }
 
-async function fetchComments({ query, token }) {
-  const authorization = token ? `Bearer ${token}` : ''
-
-  return talkAPI.get('/comments', query, { authorization })
+async function fetchComments(query) {
+  return talkAPI.get('/comments', query)
```

Comment:
This reminds me of a question I have about using `token` in some requests and not in others, and I forgot to ask in your initial Discussions PR. Why include token from `usePanoptesAuthToken()` in `useDiscussions()`, `useVotableTags()`, and` usePopularTags()`, but not in `useComments()`, `useTalkRoles()`, and `usePanoptesUsers()`? I’m in favor of not including the `token` in the SWR key when you’re not fetching user-specific data depending the signed-in user's browser session. This is probably a basic question, but good to document here on these code changes.